test/unsafe_eval/test.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import expect from '@kbn/expect';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+
+  // This test is copied from `test/api_integration/apis/general/csp.js` which
+  // runs with `--csp.unsafe_eval=false`, whereas this test runs with `--csp.unsafe_eval=true`
+  describe('csp smoke test 2', () => {
```

Comment:
Following on from my other comment --

I think we can update the rendering service jest integration tests (`src/core/server/http_resources/integration_tests/http_resources_service.test.ts`) to add a case where unsafe-eval is disabled, and then remove this separate test suite.

Reply:
Similarly to my comment above (in [`test/api_integration/apis/general/csp.js`](https://github.com/elastic/kibana/pull/124484/files/22e9efb92a2f3f6ea68c3a7ea426859b384530cf#diff-67ede3f49a9555c291dcd727bbc146cebf65d44817443f18defb791f40f473fe)), I believe there's things being tested here that we can't successfully cover in a simple unit test. But again, I might be overdoing it?

Reply:
Yeah, I think this is overkill. We should make every attempt to move towards Jest integration tests whenever we can because they are more lightweight and not as prone to flaky behavior as our full-fat functional testing suite.

Reply:
Fixed in https://github.com/elastic/kibana/pull/124484/commits/ce3a317158ad17315fbf543f9eb3e463eebec332